blockchain_connector/fabric/Makefile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Intel Corporation
```

Comment:
2018->2020

Reply:
Can the Makefiles be commoned out of the only difference lies in the wheel name.

Reply:
It can be picked during build optimization.